README.md

```diff
@@ -7,8 +7,20 @@ An operator charm providing an integrator for connecting to S3 provides.
 
 ### Deploying the S3 Integrator
 
-```bash
-$ juju deploy ./s3-integrator_ubuntu-20.04-amd64.charm
+#### Charmhub
+```shell
+juju deploy s3-integrator --channel edge
+```
+#### From source
+```shell
+git clone https://github.com/canonical/s3-integrator.git
+cd s3-integrator/
+sudo snap install charmcraft --classic
+charmcraft pack
```

Comment:
this actually requires that LXD was configured, please add:
```
lxd init --auto
lxc network set lxdbr0 ipv6.address none 
```

Reply:
Done, nice catch :)